config/main.py

```diff
@@ -12,17 +12,20 @@
 from swsssdk import ConfigDBConnector
 from natsort import natsorted
 from minigraph import parse_device_desc_xml
+from portconfig import get_port_config
 
 import aaa
 import mlnx
 
-SONIC_CFGGEN_PATH = "sonic-cfggen"
+PLATFORM_ROOT_PATH = '/usr/share/sonic/device'
+SONIC_CFGGEN_PATH = '/usr/local/bin/sonic-cfggen'
+HWSKU_KEY = 'DEVICE_METADATA.localhost.hwsku'
+PLATFORM_KEY = 'DEVICE_METADATA.localhost.platform'
```

Comment:
Why did you add definitions for `PLATFORM_ROOT_PATH`, `HWSKU_KEY` and `PLATFORM_KEY`? They are not used anywhere in the file.

Reply:
Joe, it is added as part of the initiall pull request. This is not necessary. Will remove the lines